.circleci/scripts/helm_release.sh

```diff
@@ -51,8 +51,8 @@ dependencies() {
 
 package() {
   for chart in "${CHARTS[@]}"; do
-    CHART_VERSION_CMD=$(helm inspect chart "$chart" | grep version | cut -d' ' -f2|tr -d " \t\n\r")
-    APP_VERSION_CMD=$(helm inspect chart "$chart" | grep appVersion | cut -d' ' -f2|tr -d " \t\n\r")
+    CHART_VERSION_CMD=$(echo "$CIRCLE_TAG" | cut -d "/" -f 4)
+    APP_VERSION_CMD=$(echo "$CIRCLE_TAG" | cut -d "/" -f 4)
```

Comment:
Please add a check that the version should start with a `v` to make sure that it fails the script when the tag format is changed or incorrect version is provided.